GetJobProfiles/CypherToContentRecipes/CreateSkillContentItems.recipe.json

```diff
@@ -13,7 +13,7 @@
             "name": "CypherToContent",
             "queries": [
                 {
-                    "query": "MATCH (o:esco__Skill)\r\nreturn { ContentType: 'Skill', TitlePart:{Title:o.skos__prefLabel}\r\n,GraphSyncPart:{Text:o.uri},\r\nSkill:{Description:{Html:'<p>TBC</p>'}}\r\n}\r\norder by o.skos__prefLabel\r\nskip [token:skip] limit [token:limit]"
+                    "query": "MATCH (o:esco__Skill)-[r:dct__description]-(s {esco__language: \"en\"}\r\nreturn { ContentType: 'Skill', TitlePart:{Title:o.skos__prefLabel}\r\n,GraphSyncPart:{Text:o.uri},\r\nSkill:{Description:{Html:s.esco__nodeLiteral}}\r\n}\r\norder by o.skos__prefLabel\r\nskip [token:skip] limit [token:limit]"
```

Comment:
Is this missing an `)` for the s node?

Reply:
That may well be a copy and paste fail! good spot!